src/main/scala/mesosphere/marathon/api/v2/AppsResource.scala

```diff
@@ -83,7 +93,13 @@ class AppsResource @Inject() (
       val resolvedEmbed = InfoEmbedResolver.resolveApp(embed) +
         AppInfo.Embed.Counts + AppInfo.Embed.Deployments
       val mapped = await(appInfoService.selectAppsBy(selector, resolvedEmbed))
-      Response.ok(jsonObjString("apps" -> mapped)).build()
+
+      println("+++ Send apps")
+      val stream = new AppInfoStream(mapped)
+      Response.ok(stream).build()
```

Comment:
Using the stream I constantly get `org.glassfish.jersey.server.internal.process.MappableException: org.eclipse.jetty.io.EofException`. Any ideas @timcharper?